docs-src/faq.md

```diff
@@ -1,6 +1,59 @@
 # Frequently Asked Questions (FAQ)
 
-*   Q: Will there be a default controller implementation (in this repo)?<br>
+*   **Q: Will Service API replace the Ingress API?<br>**
+    A: No. The Ingress API is GA since Kubernetes 1.19. There are no
+    plans to deprecate this API and we expect most ingress controllers
+    to support it indefinitely.
+
+*   **Q: How can I get involved with Service API?<br>**
+    A: The [community](/community) page keeps track of how to get
+    involved with the project.
+
+*   **Q: Will there be a default controller implementation (in this repo)?<br>**
     A: There is no current plan to have an "official" or "default"
     implementation. You will see the controller code in this repo be
     used for testing the support libraries.
+
+*   **Q: How can I expose custom capabilities through Service API?<br>**
+    A: There is a lot of diversity in the networking and proxying
+    ecosystem, and many products will have features that are not directly
+    supported in the API.  However, there are a few mechanisms available
+    for extending the API with implementation-specific capabilities:
+
+    * Decorate Service API objects with implementation-specific objects. A
+      policy or configuration object could match the Service API object either
+      by name or by using an explicit object reference.
+
+        For example, given a `Gateway` object with name "inbound",
+        creating a `AccessPolicy` object that also has the name "inbound"
+        could cause an implementation to attach a specified access
+        control policy. This is an example of matching the object by name.
+
+        Service API uses this decorator pattern with the
+        [`BackendPolicy`](/spec/#networking.x-k8s.io/v1alpha1.BackendPolicy)
+        resource, which modifies how a `Gateway` should forward traffic
+        to a backend target (commonly a Kubernetes `Service`). This is
+        an example of using explicit object references.
+
+    * Use implementation-specific values for string fields. In many
+      places, the fields of Service API resources have the type
+      "string". This allows an implementation to support custom values
+      for those fields in addition to any values specified in the API.
+
+    * Use implementation-specific annotations. For some kinds of
+      configuration, implementations may choose to support custom
+      annotations on Servce API objects. This approach continues
+      a proud tradition of extending Ingress objects.
+
+    * Use API-defined extension references. In a few cases, Service
+      API objects allow references to arbitrary other Kubernetes
+      objects. These are cases where a single object may need to
+      refer to some implementation-defined values multiple times
+      or in a way that is context-dependent.
+
+*  **Q: Where can I find Service API releases?<br>**
```

Comment:
Another follow up question from here:

- What does an alpha release mean? Will there be breaking changes in the next release?